doc/api/crypto.md

```diff
@@ -2082,6 +2082,20 @@ encryption mechanism, PEM-level encryption is not supported when encrypting
 a PKCS#8 key. See [RFC 5208][] for PKCS#8 encryption and [RFC 1421][] for
 PKCS#1 and SEC1 encryption.
 
+### `keyObject.equals(otherKeyObject)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `otherKeyObject`: {KeyObject} A `KeyObject` with which to
+  compare `keyObject`.
+* Returns: {boolean}
+
+Returns `true` or `false` depending on whether the keys have exactly the same
+type, value, and parameters. This method is not
+[constant time](https://en.wikipedia.org/wiki/Timing_attack).
```

Comment:
Nit: please move the link to the bottom of the document.

```suggestion
type, value, and parameters. This method is not [constant time][].
```

Reply:
There are other links like so, commonly ones that are not re-used, within the document that are also inline. I'd like to ignore this Nit to be able to land without re-running CI just for a doc change.

Reply:
Consider it optional :)